server/app/services/account.service.js

```diff
@@ -110,7 +110,7 @@ const resendConfirmationEmail = async email => {
     return {
       success: false,
       code: "REG_ACCOUNT_NOT_FOUND",
-      message: `Email ${email} is not registered. `
+      message: `Resending confirmation email to ${email} failed due to: ${err.message}`
```

Comment:
You could get to this point if

- The email does not correspond to a registered account (hence the REG_ACCOUNT_NOT_FOUND name), or
- The attempt to resend the confirmation failed for some reason
The error handling logic should probably distinguish between these possibilities, and return different codes, so the client could give more applicable instructions.